test/verify/check-storage-nfs

```diff
@@ -215,7 +215,8 @@ class TestStorageNfs(StorageCase):
         b.click("#detail-header button:contains(Unmount)")
         self.dialog_wait_open()
         b.wait_in_text("#dialog", "The filesystem is in use")
-        b.wait_in_text("#dialog", "of user root")
+        desc = b.text(".units-table td:first-child")
+        self.assertTrue(re.match(r"Session \d+ of [uU]ser root", desc))
```

Comment:
Please no assertTrue() -- failures are really useless. Use [assertRegex](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRegex) instead. Unfortunately that doesn't accept flags such as `re.I`, so the string is fine I think.

Reply:
Updated